test/services/contract_wrappers/config_wrapper.js

```diff
@@ -46,4 +46,30 @@ describe('Config Wrapper', () => {
       expect(atlasStake1CallStub).to.be.calledOnce;
     });
   });
+
+  describe('CHALLENGE_DURATION', () => {
+    const challengeDurationStub = sinon.stub();
+    const challengeDurationCallStub = sinon.stub();
```

Comment:
you should recreate the stubs on every `beforeEach` or at least reset them. Currently it only works because there is only one `it`